opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -716,3 +546,157 @@ message Exemplar {
   // or if the trace is not sampled.
   bytes trace_id = 5;
 }
+
+//
+// Add deperecated messages below this line
```

Comment:
```suggestion
// Move deprecated messages below this line
```

Reply:
Hi @Oberon00  sounds good. Will make that change once the majority of approvers decide this is useful. For those of us have  of APM/metrics background but very new to OTel metrics